schema/cam/cam_schema_2-1-0.json

```diff
@@ -126,308 +128,829 @@
+                    "value": {
+                      "$comment": "if not provided, 'value' = 161 (unavailable)",
+                      "type": "integer",
+                      "description": "unit: 0.1 m/s2. pointOneMeterPerSecSquaredForward(1), pointOneMeterPerSecSquaredBackward(-1), unavailable(161)",
+                      "default": 161,
+                      "minimum": -160,
+                      "maximum": 161
+                    },
+                    "confidence": {
+                      "$comment": "if not provided, 'confidence' = 102 (unavailable)",
+                      "type": "integer",
+                      "description": "pointOneMeterPerSecSquared(1), outOfRange(101), unavailable(102)",
+                      "minimum": 0,
+                      "maximum": 102
+                    }
+                  }
+                },
+                "performance_class": {
+                  "$comment": "if not provided, 'performance_class' = 0 (unavailable)",
                   "type": "integer",
-                  "description": "Unit: 0.1 m/s2. pointOneMeterPerSecSquared(1), outOfRange(101), unavailable(102)",
+                  "description": "unavailable 0), performanceClassA(1), performanceClassB(2), reserved(3-7)",
+                  "default": 0,
                   "minimum": 0,
-                  "maximum": 102
+                  "maximum": 7
+                },
+                "cen_dsrc_tolling_zone": {
+                  "type": "object",
+                  "description": "information about a the position of a CEN DRSC Tolling Station operating in the 5,8 GHz frequency band.",
+                  "required": [
+                    "protected_zone_latitude",
+                    "protected_zone_longitude"
+                  ],
+                  "properties": {
+                    "protected_zone_latitude": {
+                      "type": "integer",
+                      "description": "Unit: 0.1 microdegree. oneMicrodegreeNorth (10), oneMicrodegreeSouth (-10), unavailable(900000001)",
+                      "default": 900000001,
+                      "minimum": -900000000,
+                      "maximum": 900000001
+                    },
+                    "protected_zone_longitude": {
+                      "type": "integer",
+                      "description": "Unit: 0.1 microdegree. oneMicrodegreeEast (10), oneMicrodegreeWest (-10), unavailable(1800000001)",
+                      "default": 1800000001,
+                      "minimum": -1800000000,
+                      "maximum": 1800000001
+                    },
+                    "cen_dsrc_tolling_zone_id": {
+                      "type": "integer",
+                      "description": "id of a protected communication zone",
+                      "minimum": 0,
+                      "maximum": 134217727
+                    }
+                  }
+                }
+              }
+            },
+            "rsu_container_high_frequency": {
```

Comment:
`rsu_container_high_frequency` is a new item; it should go in its own commit, rather than be confalted in a commit that fixes the existing schema.

It would also make reviewing easier by splitting the changes in smaller, easier-to-digest changes.

Reply:
Ok, done.